### PR TITLE
Add mc - minio client as download option

### LINKED ISSUE #294: Include Support for Minio Client
Provide support of minio [client](https://docs.min.io/docs/minio-client-complete-guide.html) in arkade 
## Expected Behaviour

User should be able to install minio client on local using arkade
## Current Behaviour

There is no support of minio client in arkade
## Possible Solution

Append minio client in list of arkade tools

### PR DESCRIPTION
**Pull request by nitishkumar71**
Signed-off-by: Nitishkumar Singh <nitishkumarsingh71@gmail.com>

## Description

Closes #294 
## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

Installed Minio client in my local using arkade
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: pkg/get/tools.go [hunk 1]
```diff
@@ -836,12 +836,42 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
...
-		{{- end -}}`,
+			{{.Name}}_windows_amd64.exe
+			{{- else if eq .OS "darwin" -}}
+			{{.Name}}_darwin_amd64
+			{{- else -}}
+			{{.Name}}_linux_amd64
+			{{- end -}}`,
+		})
+
+	tools = append(tools,
```
Comment (alexellis) [inline pkg/get/tools.go:847]: Just curious  and I'm not asking you to change this, but do we not need, or can we not specify a version?

Comment (alexellis) [reply pkg/get/tools.go:847]: Since this is a custom example, could you add a unit test too?

Comment (nitishkumar71) [reply pkg/get/tools.go:847]: We can fix the version, but when I looked into minio doc they have strategy to place their latest release at the given url. Sure, will add unit test case for it.

Comment (alexellis) [reply pkg/get/tools.go:847]: Sounds good to me.

Comment (nitishkumar71) [reply pkg/get/tools.go:847]: Done

Comment (nitishkumar71) [reply pkg/get/tools.go:847]: @alexellis

Comment (alexellis) [reply pkg/get/tools.go:847]: Thanks

**Review by alexellis — APPROVED** [review]
LGTM, thank you!

**Comment by alexellis** [comment]
/set title: Add mc - minio client as download option

**Comment by alexellis** [comment]
@nitishkumar71 perhaps we can update the minio app to say use "arkade get mc" to download the CLI?

**Comment by nitishkumar71** [comment]
> @nitishkumar71 perhaps we can update the minio app to say use "arkade get mc" to download the CLI?

Done

**Pull request merged**